src/platforms/common/blackpill-f4/blackpill-f4.h

```diff
@@ -272,12 +274,28 @@ extern bool debug_bmp;
 #define IRQ_PRI_USBUSART     (2U << 4U)
 #define IRQ_PRI_USBUSART_DMA (2U << 4U)
 #define IRQ_PRI_TRACE        (0U << 4U)
```

Comment:
While this becomes unused in this switch-over, it's a fair note that it remains useful when we get both modes compiling into the firmware together.

Reply:
`IRQ_PRI_TRACE` will be used once PR1833 moves meaningfully, and I'd rename it to `IRQ_PRI_TRACETIM` or something to make it carry meaning related to Timer-based Manchester capture.